### PR TITLE
Add test info in LH and Stress reports

### PR DESCRIPTION
**Pull request by philipktlin**
Add test info into test reports for LH and stress test, which will send to Log analytics and used in test dashboard.

### CODE REVIEW

File: test/modules/TestAnalyzer/ModuleMessagesReport.cs [hunk 1]
```diff
@@ -41,6 +42,10 @@ public ModuleMessagesReport(string moduleId, StatusCode statusCode, long receive
 
         public IList<MissedMessagesDetails> MissedMessages { get; }
 
+        public bool IsPassed => this.StatusCode == StatusCode.AllMessages;
```
Comment (and-rewsmith) [inline test/modules/TestAnalyzer/ModuleMessagesReport.cs]: Won't this always report as passing?

File: test/modules/TestAnalyzer/Settings.cs [hunk 2]
```diff
@@ -41,7 +40,12 @@ class Settings
             this.WebhostPort = Preconditions.CheckNonWhiteSpace(webhostPort, nameof(webhostPort));
             this.ToleranceInMilliseconds = Preconditions.CheckRange(tolerance, 0);
             this.StoragePath = storagePath;
-            this.TestInfo = testInfo.Split(",", StringSplitOptions.RemoveEmptyEntries);
+            this.TestInfo = testInfo.Split(",", StringSplitOptions.RemoveEmptyEntries)
```
Comment (and-rewsmith) [inline test/modules/TestAnalyzer/Settings.cs]: Beware that StringSplitOptions might not work as you expect.
https://stackoverflow.com/questions/10682301/stringsplitoptions-removeemptyentries-doesnt-work-as-advertised/10682341

```"      " by definition is not empty (it is actually whitespace), so it is not removed from resulting array.```

**Comment by and-rewsmith** [comment]
I'm going to make the changes to upload to different log type.

File: test/modules/TestAnalyzer/Settings.cs (hunk 2 above)
Comment (philipktlin) [reply test/modules/TestAnalyzer/Settings.cs]: it should be fine since I ensure '=' should be there.

**Pull request merged**